themes/dark/assets/js/store.js

```diff
@@ -296,10 +333,20 @@ const store = new Puex({
                 }
                 Object.keys(res.data).forEach(section => {
                     const config = res.data[section];
-                    store.commit(ADD_CONFIG, { section, config });
+                    return store.commit(ADD_CONFIG, { section, config });
```

Comment:
Why're you returning inside a `forEach`?

Reply:
Good question. Cant remember changing this

Reply:
Should I just remove the `return`, or return the result of the Object.keys...?

Reply:
Remove the `return`.

Reply:
Why do we return in this method at all? It's never used.